docs/setup/upgrade/saved-objects-migration.asciidoc

```diff
@@ -3,7 +3,7 @@
 
 Each time you upgrade {kib}, an upgrade migration is performed to ensure that all <<managing-saved-objects,saved objects>> are compatible with the new version.
 
-NOTE: To help you prepare for the upgrade to 8.0.0, 7.17.0 includes an https://www.elastic.co/guide/en/kibana/7.17/upgrade-assistant.html[*Upgrade Assistant*].
+NOTE: {kib} includes an https://www.elastic.co/guide/en/kibana/7.17/upgrade-assistant.html[*Upgrade Assistant*] to help you prepare for an upgrade.
```

Comment:
Not mandatory, but you can get the first part of this URL by using the attribute from https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc?plain=1#L12
```suggestion
NOTE: {kib} includes an {kibana-ref-all}/7.17/upgrade-assistant.html[*Upgrade Assistant*] to help you prepare for an upgrade.
```